tests/codeception/functional/Navigation/NavigationDropDownsCest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+use Faker\Factory;
+/**
+ * Test for the lockup settings
+ */
+class NavigationDropDownsCest {
+
+  /**
+   * Create some content and test the dropdown menu
+   */
+  public function testDropdownMenus(FunctionalTester $I) {
+
+    $I->logInWithRole('Administrator');
+    $I->amOnPage('/admin/appearance/settings/stanford_basic');
+    $I->see('Enable dropdowns for navigation menu');
+    $I->checkOption('Enable dropdowns for navigation menu');
+    $I->click('Save configuration');
```

Comment:
How about adding this checkbox to the site settings page: `/admin/config/system/basic-site-settings`? I can't remember if site managers have access to the the theme settings. And if they don't it's not bad. We could start it off as a requested feature.